.github/workflows/integration-tests-v1.yml

```diff
@@ -33,8 +33,6 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-        with:
-          ref: ${{ inputs.branch }}
```

Comment:
will this still checkout a correct branch?

Reply:
It will now checkout the correct branch. Before `inputs.branch` was not defined at all.
When ref is omitted this defaults to the branch that triggered the workflow.